scripts/cloudlab/setup_node.sh

```diff
@@ -58,3 +59,10 @@ if [ "$SANDBOX" == "firecracker" ]; then
     $SCRIPTS/create_devmapper.sh
 fi
 
+if [ -z "$USE_STARGZ" ]; then
+    USE_STARGZ="no-stargz"
```

Comment:
having both no-* and use-* flags doesn't make sense. please leave only use-* and use negation where appropriate